CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Prelude
 
 Released on: 2018-09-13
 
+Please note that due to a critical bug identified in this release affecting container
+log collection when the `container_collect_all` was set, would lead to an agent
+dedalock. The severity of the issue has led us to remove the images from dockerhub,
+and our linux (`deb` and `rpm`) repositories. If you have upgraded to this version,
```

Comment:
let's mention again: `If you have upgraded to this version _on Linux or Docker_, please [...]`